.github/scripts/toolchain/build-cygwin.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../config.sh
```

Comment:
I think build-cygwin.sh and build-cygwin-crt.sh could be united and configured slightly different.